### ISSUE TITLE
ImportError: cannot import name 'SessionMessageBus' from 'dasbus'

### ISSUE THREAD

**Issue by jiffygist** [matrix-org/pantalaimon#186]
I'm trying to install it from latest master like this
```
python3.11 -m virtualenv venv
source venv/bin/activate
pip install .[ui]
```
However, when I run `pantalaimon` it errors with
```
Traceback (most recent call last):
  File "/home/herman/prog/build/pantalaimon/venv/bin/pantalaimon", line 5, in <module>
    from pantalaimon.main import main
  File "/home/herman/prog/build/pantalaimon/venv/lib/python3.11/site-packages/pantalaimon/main.py", line 33, in <module>
    from pantalaimon.ui import UI_ENABLED
  File "/home/herman/prog/build/pantalaimon/venv/lib/python3.11/site-packages/pantalaimon/ui.py", line 31, in <module>
    from dasbus import SessionMessageBus
ImportError: cannot import name 'SessionMessageBus' from 'dasbus' (/home/herman/prog/build/pantalaimon/venv/lib/python3.11/site-packages/dasbus/__init__.py)
```
Tried as well with python3.12 and python3.13

**Comment by chookity-pokk** [comment]
Sorry for the delay, just finished with a big move and the holidays. Can you #187?

**Comment by jiffygist** [comment]
Yeah this works

**Comment by chookity-pokk** [comment]
@jiffygist Great! I will merge that in later today and ping you when it's ready.

**Comment by chookity-pokk** [comment]
Fixed with #187
